testing/bdd/features/validation/timeBase_timeformat_constraints.feature

```diff
@@ -45,6 +45,55 @@ Feature: ttp:timeBase-related attribute constraints
     | timeBase_timeformat.xml  | clock     | 0142:05:60.8  | 145:00:47     |               |
     | timeBase_timeformat.xml  | clock     |               |               | 199:00:60.4   |
 
+  # SPEC-CONFORMANCE: R55 R56 R58 R59
+  Scenario: Valid times according to timeBase in div
+    Given an xml file <xml_file>
+    When it has timeBase <time_base>
+    And it has div begin time <div_begin>
+    And it has div end time <div_end>
+    Then document is valid
+
+    Examples:
+    | xml_file                | time_base | div_begin    | div_end        |  
+    | timeBase_timeformat.xml | clock     | 15.58m       | 1.5h           |  
+    | timeBase_timeformat.xml | clock     | 42:05:60.8   | 45:00:47       |  
+    | timeBase_timeformat.xml | media     | 42:05:60.8   | 45:00:47.0     |  
+    | timeBase_timeformat.xml | media     | 67.945s      | 125.0s         |  
+    | timeBase_timeformat.xml | media     | 999:09:60.8  | 1999:00:60.999 |  
+    | timeBase_timeformat.xml | clock     | 45:00:47     | 1400h          |  
+    | timeBase_timeformat.xml | clock     | 1400h        |                |  
```

Comment:
NB this reads like '2 o'clock in the afternoon' but really means 'one thousand four hundred hours after the epoch'!